src/App.tsx

```diff
@@ -34,7 +34,12 @@ const App: React.FC = () => {
     const { StatusBar } = Plugins;
     StatusBar.setBackgroundColor({color: "#333333"});
 
-    ScreenOrientation.lock(ScreenOrientation.ORIENTATIONS.PORTRAIT);
+    try {
+      ScreenOrientation.lock(ScreenOrientation.ORIENTATIONS.PORTRAIT);
+    } catch (err) {
+      console.log(err);
```

Comment:
Did you tested?

Reply:
It cannot work. isn't it inside an async await. So you should resolve the returning promise in the normal whay then/catch